docker/Dockerfile

```diff
@@ -1,51 +1,22 @@
-# ----------------
-# STEP 1:
-FROM node:erbium-alpine AS build-z2m
+FROM node:12.20.1-buster
+# removing the dependency on fibers would mean we could go for an image without the compilers+python, also allow us to get beyond node 14
```

Comment:
It is needed in the webpack build

Reply:
Yeah. It's an obsolete module and a bit ugly. It is possible apparently to remove though may incur performance hit